test/invariant/fuzz/Setup.t.sol

```diff
@@ -79,5 +91,101 @@ contract Setup is Actors {
                 registry.getProfileById(_id).anchor
             );
         }
+
+        // Create pools for each strategy
+        _initPools();
+    }
+
+    function _initPools() internal {
+        defaultRegistrationStartTime = uint64(block.timestamp);
+        defaultRegistrationEndTime = uint64(block.timestamp + 7 days);
+        defaultAllocationStartTime = uint64(block.timestamp + 7 days + 1);
+        defaultAllocationEndTime = uint64(block.timestamp + 10 days);
+        defaultWithdrawalCooldown = 1 days;
+        DEFAULT_MAX_BID = 1000;
+
+        for (uint256 i = 1; i <= uint256(type(PoolStrategies).max); i++) {
+            address _deployer = _ghost_actors[i % 4];
```

Comment:
why is it `% 4 `? if we are interested to use a specific address we could mark it explicitly just in case `_ghost_actors` is changed in the future

Reply:
the first 4 actors are anchor owner, no?

Reply:
makes sense